tests/fixtures/xfail/excessive-indentation.rst

```diff
@@ -0,0 +1,75 @@
+.. expect: 15: Excessive indentation in nested section (excessive-indentation)
+.. expect: 20: Excessive indentation in nested section (excessive-indentation)
+.. expect: 26: Excessive indentation in nested section (excessive-indentation)
+.. expect: 30: Excessive indentation in nested section (excessive-indentation)
+.. expect: 35: Excessive indentation in nested section (excessive-indentation)
+.. expect: 42: Excessive indentation in nested section (excessive-indentation)
+.. expect: 49: Excessive indentation in nested section (excessive-indentation)
+.. expect: 51: Excessive indentation in nested section (excessive-indentation)
+.. expect: 53: Excessive indentation in nested section (excessive-indentation)
+.. expect: 57: Excessive indentation in nested section (excessive-indentation)
+.. expect: 71: Excessive indentation in nested section (excessive-indentation)
+
+The most common mistakes is indenting lists and directives under a paragraph:
+
+  * This list shouldn't be indented
+  * Otherwise it's rendered inside a blockquote
+
+Directives also shouldn't be indented under a paragraph:
+
+  .. note:: like this one
+
+
+
+* Nested lists should be indented properly
+
+    * the bullet of this list should have been under the N
+
+* Same goes for directives
+
+    .. note:: this should have been under the S
+
+
+.. note::
+
+     * the opposite is also true
+     * lists nested under directives
+     * should be indented properly
+     * (but maybe they don't have to?)
+
+.. note::
+
+     .. note:: this is also not allowed atm, but maybe it should
+
+
+There are also other types of lists:
+
+* bullet lists
+
+    - bullet lists with different bullets
+
+        1. numbered lists
+
+             2) more numbered lists
+
+                (3) numbered lists with more parentheses
+
+                      #. autonumbered lists
+
+
+Each of these should give an error, since they are all indented wrong.
+
+Numbered lists that start with letters (a. b. c. ...)
+and roman numerals (I. II. III. ...) are only supported by Sphinx 7+
+so we just ignore them
+
+
+This is a known false positive:
+
+   >>> import sys
+   >>> sys.stdout.writelines(result)
+       1. Beautiful is better than ugly.
+   -   2. Explicit is better than implicit.
+   -   3. Simple is better than complex.
+   +   3.   Simple is better than complex.
+   ?     ++
```

Comment:
I found this in `library/difflib.rst` in the CPython docs, and it fails on line 71, on the indented `1.`.  This shouldn't happen, so the checker should be fixed to ignore the content of code blocks.

Reply:
Is this really a code block without the double colon? According to [the spec](https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#block-quotes) it should be a block quote, but according to the [html outout](https://docs.python.org/3/library/difflib.html#differ-example) it looks like a code-block. Strange.

If detected as a code block by sphinx-lint it is hidden from your test (see `git grep rst_only`).

Reply:
Indeed the problem was the missing double colon.  After adding that and using `hide_non_rst_blocks` sphinx-lint skipped the block and avoided the false-positive.